modules/cnf-topology-aware-lifecycle-manager-about-cgu-crs.adoc

```diff
@@ -33,218 +45,373 @@ After {cgu-operator} completes a cluster update, the cluster does not update aga
 
-* Blocking CRs are missing from the system
-* Blocking CRs have not yet finished
+* `true`
++
+The pre-caching spec is valid and consistent.
+* `false`
++
+The pre-caching spec is incomplete.
 
-[id="upgrade_not_completed"]
-== The UpgradeNotCompleted state
+{cgu-operator} uses the `PrecachingSucceeded` condition to report status information as follows:
 
-In the `UpgradeNotCompleted` state, {cgu-operator} enforces the policies following the remediation plan defined in the `UpgradeNotStarted` state.
+* `true`
++
+TALM has concluded the pre-caching process. If pre-caching fails for any cluster, the update fails for that cluster but proceeds for all other clusters. A message informs you if pre-caching has failed for any clusters.
+* `false`
++
+Pre-caching is still in progress for one or more clusters or has failed for all clusters.
+
+For more information see the "Using the container image pre-cache feature" section.
+
+[id="creating_backup"]
+== Creating a backup
+For {sno}, {cgu-operator} can create a backup of a deployment before an update. If the update fails, you can recover the previous version and restore a cluster to a working state without requiring a reprovision of applications. To use the backup feature you first create a `ClusterGroupUpgrade` CR with the `backup` field set to `true`. To ensure that the contents of the backup are up to date, the backup is not taken until you set the `enable` field in the `ClusterGroupUpgrade` CR to `true`.
+
+{cgu-operator} uses the `BackupSucceeded` condition to report the status and reasons as follows:
+
+* `true`
++
+Backup is completed for all clusters or the backup run has completed but failed for one or more clusters. If backup fails for any cluster, the update fails for that cluster but proceeds for all other clusters.
+* `false`
++
+Backup is still in progress for one or more clusters or has failed for all clusters.
+
+For more information, see the "Creating a backup of cluster resources before upgrade" section.
+
+[id="updating_clusters"]
+== Updating clusters
+{cgu-operator} enforces the policies following the remediation plan.
+Enforcing the policies for subsequent batches starts immediately after all the clusters of the current batch are compliant with all the managed policies. If the batch times out, {cgu-operator} moves on to the next batch. The timeout value of a batch is the `spec.timeout` field divided by the number of batches in the remediation plan.
+
+{cgu-operator} uses the `Progressing` condition to report the status and reasons as follows:
+
+* `true`
++
+{cgu-operator} is remediating non-compliant policies.
+* `false`
++
+The update is not in progress. Possible reasons for this are:
++
+** All clusters are compliant with all the managed policies.
+** The update has timed out as policy remediation took too long.
+** Blocking CRs are missing from the system or have not yet completed.
 
```

Comment:
** CGU is not enabled
** Backup is still in progress

Reply:
ok